CHANGELOG.md

```diff
@@ -3,6 +3,17 @@
 This changelog goes through all the changes that have been made in each release on the
 [vocascan-frontend](https://github.com/vocascan/vocascan-frontend).
 
+## [v1.2.1](https://github.com/vocascan/vocascan-frontend/releases/tag/v1.2.1) - 2022.01.29
+
+This release of vocascan-frontend fixes the country flags in firefox and the linebreaks in the vocab card description. The new password complexity indicator will force secure passwords to the users.
+
+- Features
+  - Password complexity (#97)
+- Bugfixes
+  - fixed flags in firefox (#95)
```

Comment:
Please be consistent. Use either `fixed` or `fix`, but not mixed tenses

Reply:
I also struggled about that. Thats due to the commit names. We should generally consider what we should use next time. If I change those they wouldnt match with the commit and pr name.

Reply:
Ok, I see. Then we will leave it for now and think about naming conventions for the next PRs